beep/featurize.py

```diff
@@ -347,40 +347,44 @@ def get_rpt_dQdV_features(cls, processed_cycler_run, diag_ref=0, diag_nr=1, char
         return 1 + (peak_fit_df - peak_fit_df_ref) / peak_fit_df_ref
 
     @classmethod
-    def get_hppc_features(cls, processed_cycler_run, diag_pos=1, soc_window=7):
+    def get_hppc_features(cls, processed_cycler_run, diag_pos=1, soc_window=8):
```

Comment:
@cx26 What happens if I specify an soc window that does not exist?

Reply:
That is a good point! For now, the function will crash, giving you errors, and same thing will happen to the resistance function. I tried generating features for all the cells at the second diagnostic cycle [37], and it works for all of them. However, if we go one cycle forward [142], I am not sure if we will get errors. I can modify my code to make it fail silently in the future if you want to generate features at 142 and beyond.

Reply:
I would actually just add a quick note in the documentation? You can basically just specify the range of values that are acceptable. and what happens when that window does not exist